src/js/api/errors.js

```diff
@@ -78,10 +78,10 @@ export const FLASH_MEDIA_ERROR = 214000;
  * @param {code} [ErrorCode] - The error code.
  * @param {sourceError} [Error] - The lower level error, caught by the player, which resulted in this error.
  */
-export class PlayerError extends Error {
+export class PlayerError {
```

Comment:
What is the reasoning behind this change ? We want inheritance to leverage existing error features like stack trace, and we want a consistent interface to make our errors compatible with any technologies that handle javascript errors.

Reply:
It's because of the way `program-listeners.js` handles provider events. `error.message` isn't enumerable and is lost when the event is created via `Object.assign`; we don't want to change this (to use `Object.create`, for example) because we will potentially change our API.

Reply:
Disregard. If we avoid passing message via `super` we're allowed to define an enumerable `message` property on our `PlayerError`.